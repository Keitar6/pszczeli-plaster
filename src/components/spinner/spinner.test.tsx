import ComponentsRenderer from "react-test-renderer";
import Spinner from "./spinner.component";

describe("Spinner", () => {
  test("spinner", () => {
    const spinner = ComponentsRenderer.create(<Spinner />).toJSON();
    expect(spinner).toMatchSnapshot();
  });
});
