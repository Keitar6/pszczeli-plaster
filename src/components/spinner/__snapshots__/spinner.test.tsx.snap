// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner spinner 1`] = `
.c0 {
  height: 60vh;
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  display: inline-block;
  width: 50px;
  height: 50px;
  border: 3px solid rgba(195,195,195,0.6);
  border-radius: 50%;
  border-top-color: #636767;
  -webkit-animation: spin 1s ease-in-out infinite;
  animation: spin 1s ease-in-out infinite;
  -webkit-animation: kqAfnK 1s ease-in-out infinite;
}

<div
  className="c0"
>
  <div
    className="c1"
  />
</div>
`;
